cmd/login.go

```diff
@@ -32,7 +32,7 @@ import (
 var loginCmdArg = loginCmdArgs{tenantID: common.DefaultTenantID}
 
 var loginNotice = "'azcopy %s' command will be deprecated starting release 10.22. " +
```

Comment:
You could probably update this to 10.23 since we are skipping this release, right?

Reply:
I'll add that as a different review. Deviates from topic.